crates/tables/src/lib.rs

```diff
@@ -151,10 +169,14 @@ tables!(
         val data_plane_id: models::Id,
         // Most recent publication ID of this capture.
         val last_pub_id: models::Id,
+        // Most recent build ID of this capture
+        val last_build_id: models::Id,
         // Model of the capture as-of `last_pub_id`
         val model: models::CaptureDef,
         // Built specification of this capture as-of `last_pub_id`.
         val spec: proto_flow::flow::CaptureSpec,
+        // Hash of the last_pub_ids of all the dependencies that were used to build the capture
+        val dependency_hash: Option<String>,
```

Comment:
My thinking for using a string instead of a u64 is that it's more obviously something that should be treated as opaque.

The Option just seemed convenient as a way to represent a spec that has no dependencies. For example, a collection, or a capture with no bindings.

I don't feel especially strongly about either of those things, and could get behind a u64, where a `0` means it has no dependencies. It's convenient to have a simple constant representing "no dependencies", though, because for example when we validate collection specs, we know that they _can't_ have dependencies and so don't consult the `Dependencies::compute_hash` function at all. That sound good to you?

Reply:
> The Option just seemed convenient as a way to represent a spec that has no dependencies. For example, a collection, or a capture with no bindings.

I suppose I don't see "zero bindings" as different from "one or more bindings". It's all the same continuum, an empty set of bindings can be represented in the same hash space as non-empty ones, and thus why jump through the extra Some-check hoops to model the distinction. Or, if it remains a string, an empty string could also serve as the current None.

I've said my piece, though, and it's so minor I'm okay any way you want to roll, including merging as-as.

Reply:
That's a fair point. I'm inclined to just merge as-is, though, as it doesn't seem like we'll actually regret the current representation.